cfgov/v1/wagtail_hooks.py

```diff
@@ -219,3 +221,14 @@ class MegaMenuModelAdmin(ModelAdmin):
 
 
 modeladmin_register(MegaMenuModelAdmin)
+
+
+@receiver(post_save, sender=MegaMenuItem)
+def clear_mega_menu_cache(sender, instance, **kwargs):
+    from django.core.cache import caches
+    try:
```

Comment:
What's the rationale for this `try/except`? Is it to handle the case where the cache isn't defined?

To simplify, how would you feel about always defining the cache in local settings, but defaulting it to use [`DummyCache`](https://docs.djangoproject.com/en/2.0/topics/cache/#dummy-caching-for-development) unless the environment variable is set? This would simplify the application logic.